### PR TITLE
[Task 25] initial blockchain oracles research and draft

### PR DESCRIPTION
**Pull request by jamesylgan**
currently missing detail on impls and approaches

### CODE REVIEW

File: content/oracles.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    ]}
+/>
+
+It is typically nearly impossible to reliably determine and verify the results of many things that are used for gambling. For example, a game as simple as blackjack could be rigged in favor of the dealer to the tune of 1% gambling volume per day. With a smart contract, online gambling can be verifiably fair with transparency around any dealer biases present in games.
+
+However, many areas of online gambling are not as simple as blackjack. In sports betting, without an oracle, you cannot determine which team won a game; as a result, sports betting requires some sort of oracle. For a game like the Superbowl, **these contracts may begin to yield tens or hundreds of millions of dollars, creating a massive financial incentive for oracles to behave dishonorably, reporting data that would lead to their bets succeeding.**
+
+At this point, hopefully it is clear why we need oracles: without them, smart contracts are restricted to use cases that solely involve data that already exists on-chain, such as cryptocurrency. Once we have an oracle, we also need to consider if it is reliable enough for the use case at hand.
+
+##### Gambling Legality & Blockchain
```
Comment (rishub) [inline content/oracles.mdx]: its hard to tell this is a title when reading through, maybe just make it level 4?
```suggestion
#### Gambling Legality & Blockchain
```

File: content/oracles.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+Taking into consideration the why smart contracts and Blockchain networks might want to use oracles, here's how we would define these requirements:
+
+1. What degree of BFT does the oracle have?
+
+  * What would it cost an actor to push specific data through that oracle?
+
+2. What is the ultimate source of the data? (Company? Web feed? Sensors?)
+2. What is the cost of getting data?
+2. What is the delay in getting data?
+2.
```
Comment (rishub) [inline content/oracles.mdx]: extra bullet?

Comment (jamesylgan) [reply content/oracles.mdx]: It's mostly a placeholder to remind me to be adding extra bullets, if I think of relevant stuff to add

File: content/oracles.mdx (hunk 1 above)
Comment (jamesylgan) [reply content/oracles.mdx]: Sounds good to me - :)

**Comment by jamesylgan** [comment]
Merge if you think it looks good @rishub :)

**Pull request merged**